README.md

```diff
@@ -180,15 +180,17 @@ receivers:
 ## API
 
 The current Alertmanager API is version 2. This API is fully generated via the
-[OpenAPI
-project](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md)
+[OpenAPI project](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md)
 and [Go Swagger](https://github.com/go-swagger/go-swagger/) with the exception
 of the HTTP handlers themselves. The API specification can be found in
 [api/v2/openapi.yaml](api/v2/openapi.yaml). A HTML rendered version can be
-accessed
-[here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/prometheus/alertmanager/master/api/v2/openapi.yaml).
-Clients can be easily generated via any OpenAPI generator for all major
-languages.
+accessed [here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/prometheus/alertmanager/master/api/v2/openapi.yaml).
+Clients can be easily generated via any OpenAPI generator for all major languages.
+
+Note: Under the default config endpoints are accessed under a `/api/v1` or `/api/v2` prefix.
```

Comment:
May I suggest a small edit?

"With the default config, endpoints are accessed..."